docs/dev/stream/python.md

```diff
@@ -24,6 +24,9 @@ specific language governing permissions and limitations
 under the License.
 -->
 
+*Note: This API is based on Jython, which is not a full Python replacement and may restrict
```

Comment:
You can actually create neat looking `attention` boxes, like so `<span class="label label-danger">Attention</span>`, as seen here: https://ci.apache.org/projects/flink/flink-docs-master/dev/event_timestamps_watermarks.html#assigning-timestamps